addons/mail/static/tests/time_model_tests.js

```diff
@@ -0,0 +1,47 @@
+/** @odoo-module **/
+
+import { insertAndReplace } from '@mail/model/model_field_command';
+import { afterEach, beforeEach, start } from '@mail/utils/test_utils';
+
+QUnit.module('mail', {}, function () {
+QUnit.module('Time Model', {
+    async beforeEach() {
+        await beforeEach(this);
+
+        this.start = async params => {
+            const res = await start({ ...params, data: this.data });
+            const { apps, env, widget } = res;
+            this.apps = apps;
+            this.env = env;
+            this.widget = widget;
+            return res;
+        };
+    },
+    afterEach() {
+        afterEach(this);
+    },
+});
+
+QUnit.test('everyMinuteIntervalId field on Time model is correctly computed', async function (assert) {
```

Comment:
this should probably be 2 or 3 tests with single assert: one that it is empty (by default), one that it is not empty when used,
and maybe one more for the transition from used to not used